source/docs/software/commandbased/what-is-command-based.rst

```diff
@@ -115,27 +78,9 @@ controlling the robot drive, or keeping an arm held at a setpoint.
 
    scheduler control flow diagram
 
-When a command is scheduled, its ``initialize()`` method is called once.
-Its ``execute()`` method is then called once per call to
-``CommandScheduler.getInstance().run()``. A command is un-scheduled and
-has its ``end(boolean interrupted)`` method called when either its
-``isFinished()`` method returns true, or else it is interrupted (either
-by another command with which it shares a required subsystem, or by
-being canceled).
+When a command is scheduled, its ``initialize()`` method is called once. Its ``execute()`` method is then called once per call to ``CommandScheduler.getInstance().run()``. A command is un-scheduled and has its ``end(boolean interrupted)`` method called when either its ``isFinished()`` method returns true, or else it is interrupted (either by another command with which it shares a required subsystem, or by being canceled).
 
 Command groups
 --------------
 
-It is often desirable to build complex commands from simple pieces. This
-is achievable by
-`composing <https://en.wikipedia.org/wiki/Object_composition>`__
-commands into “command groups.” A command group is a command that
-contains multiple commands within it, which run either in parallel or in
-sequence. The command-based library provides several types of command
-groups for teams to use, and users are encouraged to write their own, if
-desired. As command groups themselves implement the ``Command``
-interface, they are `recursively
-composable <https://en.wikipedia.org/wiki/Object_composition#Recursive_composition>`__
-- one can include command groups *within* other command groups. This
-provides an extremely powerful way of building complex robot actions
-with a simple library.
+It is often desirable to build complex commands from simple pieces. This is achievable by `composing <https://en.wikipedia.org/wiki/Object_composition>`__ commands into “command groups.” A command group is a command that contains multiple commands within it, which run either in parallel or in sequence. The command-based library provides several types of command groups for teams to use, and users are encouraged to write their own, if desired. As command groups themselves implement the ``Command`` interface, they are `recursively composable <https://en.wikipedia.org/wiki/Object_composition#Recursive_composition>`__ - one can include command groups *within* other command groups. This provides an extremely powerful way of building complex robot actions with a simple library.
```

Comment:
Is there a code example of this? Might want to link to the corresponding document?

Reply:
Yeah, I'll add a link.